manifests/katello.pp

```diff
@@ -18,6 +18,12 @@
   $candlepin_consumer_summary     = "Subscription-manager consumer certificate for Katello instance ${::fqdn}"
   $candlepin_consumer_description = 'Consumer certificate and post installation script that configures rhsm.'
 
+  class { '::trusted_ca': }
```

Comment:
Can this be done via `include ::trustedca` since we aren't setting any parameters?